src/Riok.Mapperly.Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,9 @@
 #nullable enable
+Riok.Mapperly.Abstractions.EnumNamingStrategy
```

Comment:
You can move this to the shipped file directly. Mapperly only uses the shipped file. Git is used for version / shipped / unshipped tracking.

Reply:
Yes, I have done that.